toolbox/githubctl/main.go

```diff
@@ -38,37 +41,13 @@ var (
 	tag                   = flag.String("tag", "", "Tag of the release candidate")
 	releaseOrg            = flag.String("rel_org", "istio-releases", "GitHub Release Org")
 	gcsPath               = flag.String("gcs_path", "", "The path to the GCS bucket")
+	skip                  = flag.String("skip", "", "comma separated list of jobs to skip")
 	maxCommitDepth        = flag.Int("max_commit_depth", 200, "Max number of commits before HEAD to check if green")
 	maxRunDepth           = flag.Int("max_run_depth", 500, "Max number of runs before the latest one of which results are checked")
 	maxConcurrentRequests = flag.Int("max_concurrent_reqs", 50, "Max number of concurrent requests permitted")
 	githubClnt            *u.GithubClient
 	ghClntRel             *u.GithubClient
-	// unable to query post-submit jobs as GitHub is unaware of them
-	// needs to be consistent with prow config map
-	postSubmitJobsMap = map[string][]string{
-		"master": {
-			"e2e-mixer-no_auth",
-			"e2e-bookInfoTests-envoyv2-v1alpha3",
-			"istio-pilot-e2e-envoyv2-v1alpha3",
-			"e2e-simpleTests",
-			"e2e-dashboard",
-			"istio-postsubmit",
-		},
-		"release-1.0.0-snapshot-0": {
-			"e2e-mixer-no_auth",
-			"e2e-bookInfoTests-envoyv2-v1alpha3",
-			"istio-pilot-e2e-envoyv2-v1alpha3",
-			"e2e-simpleTests",
-			"e2e-dashboard",
-			"istio-postsubmit",
-		},
-		"release-0.8": {
-			"istio-postsubmit",
-			"e2e-suite-rbac-no_auth",
-			"e2e-suite-rbac-auth",
-			"e2e-cluster_wide-auth",
-		},
-	}
+	postSubmitJobs        = []string{}
```

Comment:
remove this global and pass it around. otherwise lgtm

Reply:
Done